husky_description/urdf/decorations.urdf.xacro

```diff
@@ -43,21 +43,23 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
       <child link="top_chassis_link" />
     </joint>
 
-    <!-- Spawn user rails -->
-    <link name="user_rail_link">
-      <visual>
-        <geometry>
-          <mesh filename="package://husky_description/meshes/user_rail.dae" />
-        </geometry>
-      </visual>
-    </link>
-
-    <!-- Attach user rails to base link -->
-    <joint name="user_rail" type="fixed">
-      <origin xyz="0.272 0 0.245" rpy="0 0 0" />
-      <parent link="base_link" />
-      <child link="user_rail_link" />
-    </joint>
+    <xacro:if value="$(optenv HUSKY_USER_RAILS true)">
```

Comment:
@tonybaltovski @paulbovbel 

Note this will have a pretty big affect I imagine.